src/components/inline/Button/Button.js

```diff
@@ -3,18 +3,59 @@ import React from "react";
 
 import s from "./Button.module.css";
 
-const Button = ({ children }) => (
-  <button className={s.button} type="button">
-    {children}
-  </button>
-);
+const Button = ({ button }) => {
+  let buttonView;
 
-Button.propTypes = {
-  children: PropTypes.node,
+  if (button.externalLink) {
+    buttonView = (
+      <a rel="noreferrer" target="_blank" href={button.externalLink}>
+        <button type="button" className={s.button}>
+          {button.text}
+        </button>
+      </a>
+    );
+  } else if (button.internalLink) {
+    buttonView = (
+      <a href={button.internalLink}>
```

Comment:
Internal links should use Gatsby's `<Link>` component, which is integrated with Gatsby's routing system to navigate to a separate page without triggering a page refresh. You'll need to figure out how to apply custom styles to a Gatsby `<Link>`.